src/utils/tap/getTapPath.js

```diff
@@ -0,0 +1,19 @@
+const { getGlobalConfig } = require('KegUtils/globalConfig/getGlobalConfig')
```

Comment:
This file should be moved to `clli-utils` repo along with the others

Reply:
updated